etc/config/c++.amazon.properties

```diff
@@ -3390,6 +3390,18 @@ libs.libassert.versions.10.path=/opt/compiler-explorer/libs/libassert/v1.0/inclu
 libs.libassert.versions.11.version=1.1
 libs.libassert.versions.11.path=/opt/compiler-explorer/libs/libassert/v1.1/include
 
+libs.libbpf.name=libbpf
+libs.libbpf.versions=100:122
+libs.libbpf.liblink=libbpf:dl:m
```

Comment:
I suspect the `.so` file is just called `libbpf.so`, which means this should just be `bpf:dl:m`

Reply:
Sounds good, this is what my system has installed:
```
$ ls /usr/lib64/libbpf*
/usr/lib64/libbpf.a   /usr/lib64/libbpf.so.1      /usr/lib64/libbpftracearch.so  /usr/lib64/libbpftraceparser.so     /usr/lib64/libbpftraceruntime.so
/usr/lib64/libbpf.so  /usr/lib64/libbpf.so.1.2.2  /usr/lib64/libbpftraceast.so   /usr/lib64/libbpftraceresources.so  /usr/lib64/libbpftrace.so
```